Rescue downed Efreti crew and salvage ship (from @williaji) (#8763)
endless-sky/endless-sky: data/korath/korath missions.txt

		attributes "efret"
	on offer
		event "label korath space"

> 		log "Factions" "Korath" `The Korath are reptilian, bipedal aliens whose territory is almost entirely in ruins: cities destroyed and entire planets scarred by what appears to have been a vicious civil war. Several factions of Korath survive. The "Kor Efreti" live in the shelter of a Quarg ringworld and are on peaceful terms with the Quarg. Other Korath, living as exiles near the galactic core, have been raiding human worlds to steal food and supplies. And deeper in Korath space, two massive armies of automated war drones, known as the Kor Mereti and the Kor Sestor, are locked in an unending struggle. They attack any visitors to their regi of space, including the Quarg.`

Author: I already fixed that, shortly after you started your review.

			choice
				`	"I'll go help, then."`
					accept
				`	"I won't risk my ship for the Korath."`
					goto careless

			label careless
			`	The Quarg is silent for some time. Eventually it responds, "Life is rare and precious across the vastness of space. Look about the Korath worlds and see what remains when one cares not what others suffer."`
				decline

	on stopover
		conversation
			`It takes you some time to find the <EDC-ship>. You had expected a beacon of some sort but they are either keeping radio silence to avoid drawing the Mereti or there were no survivors.`
			`	When you manage to open the hatch, you find yourself staring down a thermal rifle. The Korath shouts something at you and you hold your hands out palms forward. He limps back a few steps but keeps the weapon trained on you. You slowly pull your communicator out and play your ship's recording of the Quarg transmission. When the Korath hears the Quarg's voice they become excited. "<planet>?" he asks repeatedly. You nod your head and show your palms again.`

> 			`	He leads you into the ship, shouting into a communication device that is then picked up on speakers throughout the ship. You can only glance around as you move through the engineering section but it's clear they are running on emergency power and their Systems Core was destroyed leaving them stranded.`

Author: Oh! That's a problem. I gave them a systems core. Better fix that...

			`	The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree with more than half incapacitated or unconscious. Even if the ship is spaceworthy, there aren't enough crew to fly it.`

> 			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture and shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`

Reviewer: ```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`
```

				to display
					"flagship crew" > 1

> 			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture and shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer them to <ship>.`

Reviewer: ```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer them to the <ship>.`
```

			`	While your crew works on refreshing the atmosphere with outside air, you gather the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
				to display
					"flagship crew" == 2
			`	One by one, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
				to display
					"flagship crew" < 2
			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
				to display
					"flagship crew" > 1

> 			`	As a precaution, you make a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`

Reviewer: Is it intentional that `You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.` is not included in this line?

Author: Why would it be on that line? Are you asking to merge the paragraphs?

Reviewer: Sorry, I misunderstood what the syntax was doing here.

				to display
					"flagship crew" <= 1

	on complete
		conversation
			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch. You count the coffins together and show them the boxes of possessions.`
			`	The officer thanks you and hands you a data chip that vaguely reminds you of a license.`
			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, you presume their family.`

> 			`	You go back inside your ship rather than intrude on their grief. You slot the data chip into your systems but the information is in Korathi. You set it aside and work on a salvage plan.`

Reviewer: Correct me if I'm wrong but aren't there several Korath languages, rather than just one 'Korathi' language?

Author: The player would not recognize the specific Korath language at this point, so I just said "an alien language."

Author: I did this:

> You slot the data chip into your systems but the information is in **an alien language**.

			choice
				`	(Fly the <EDC-ship> home)`
					goto runforit
				`	(Abandon the ship)`
					goto abandon
			
			label abandon
			action
				fail
			`	Your crew abandons the Korath ship to the elements. It's just one more tragic story amid the ruins of their civilization.`
				flee
			
			label runforit
			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
				launch

	on complete
		conversation
			action
				# Ship name should match the <EDC-ship> substitution above.
				give ship "Charm-Shallop (Distressed)" "Esketari"
			`There are no crowds waiting for you when you land; rather, the ground crews seem astonished to see the ship. It takes you some time to arrange for repairs. By the time you're finished, you see a pavilion has been set up near the <EDC-ship> and small groups of Korath periodically stop by.`
			choice
				`	(Check it out.)`
				`	(I've got places to be.)`
					decline
				

> 			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand. The drink is fermented and has a sweet and sour taste that leaves you lips and mouth tingling and then numb. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`

Reviewer: I would go with Warp's suggestion to give the player a choice to refuse the drink.

Author: You can now `(Make a motion as if drinking.)`

	on visit
		dialog "You arrive, but you notice the <EDC-ship> is absent. You'll have to bring it here to finish the salvage mission."
	npc save accompany
		# Ship name should match the <EDC-ship> substitution above.
		government "Kor Efret"
		personality timid escort
		ship "Charm-Shallop (Distressed)" "Esketari"

mission "Efreti: Distress Call 6"
	name "Translate Title"
	description "Translate the Korath data chip."
	source "Kuwaru Efreti"
	invisible
	landing

	to offer
		has "Efreti: Distress Call 5: done"
	on offer
		conversation
			`The Quarg have translated the Korath language for you in the past. Would you like to see if they will tell you what the data chip says?`
			choice
				`	(Ask the Quarg to translate the data chip.)`
				`	(Maybe another time.)`
					defer

> 				`	(I really don't care what it says.)`

Reviewer: ```suggestion
				`	(I don't care what it says.)`
```

			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`
				to display
					"flagship crew" > 1
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer them to the <ship>.`
				to display
					"flagship crew" <= 1
	
	on complete
		payment 300000
		conversation
			`The Efreti commander hails the authorities once you enter the system and they have medical crews waiting when you land. It takes several hours to evacuate the injured as doctors or maybe paramedics swarm over them to triage, treat, and prepare for transport. The three who died during the flight are treated with respect and carried off the <ship>.`

> 			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`

Reviewer: ```suggestion
			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave their dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

> 			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to seek out those where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`

Reviewer: ```suggestion
			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to find those where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

Author: No, this changes the meaning. "Seek out" just means you're trying to find it. "Find" means you found it. The Quarg are expressing the hopelessness of the search. Those Efret are looking for things they know they won't find, because they already know their worlds are dead.

Reviewer: How about this?
```suggestion
			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to search for those ruins where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

Author: They're searching for worlds to live on, and ruins in which to salvage. That run-on sentence is confusing.

Author: How about this?

> The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again.

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

substitutions
	"<EDC-ship>" "Esketari"

mission "Efreti: Distress Call 1"
	minor
	landing
	name "Distress call from the <EDC-ship>"
	description "The Quarg have asked you investigate a distress call from the Efreti ship <EDC-ship> which has crashed on <destination> with <bunks> survivors."
	source "Karek Fornati"
	destination "Seleptra Nak"

> 	passengers 40

Author: An excellent point. I will test this in 20-30 hours when I have my normal test machines again.

> 	passengers 40

Author: I implemented it. If you forget the bunks, you get a `blocked` message.

> blocked `Looking towards the downed <EDC-ship>, you suddenly realize you forgot the <bunks> bunks you need for the evacuees.`

> 	blocked "A Quarg ship broadcasts that they need a ship with at least <bunks> bunks for an emergency evacuation."

Reviewer 2: Can't the Quarg send their own ships to help? If they're right there, then they should be able to help some.

			`	The Korath shouts something at you, and you hold your hands out with your palms forward. He limps back a few steps, but keeps the weapon trained on you. Slowly pulling a communicator out, you play a recording of the Quarg transmission you received earlier. Once the Korath hears the Quarg's voice, he becomes excited and lowers his rifle. "<planet>?" he asks repeatedly. You nod your head and show your palms again.`
			`	He leads you into the ship, shouting into a communication device that is then broadcasted on speakers throughout the vessel. You can only glance around as you move through the engineering section, but it's clear they are running on emergency power, and several important-looking systems are destroyed. The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree, while more than half are incapacitated or unconscious. Even if the ship is spaceworthy, there aren't enough crew to fly it.`

> 			`	Quickly count the number of survivors, you communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms, and makes quick circles. You think he is agreeing and wants to hurry. Calling your crew over to help, the survivors are moved onboard the <ship>. Within a couple of hours you are ready to go.`

Reviewer: ```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms, and makes quick circles. You think he is agreeing and wants to hurry. Calling your crew over to help, the survivors are moved onboard the <ship>. Within a couple of hours you are ready to go.`
```

				to display
					"flagship crew" > 1
				accept
			# No "to display" is required here due to the "accept" above:

> 			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer them to the <ship>.`

Reviewer: ```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer the survivors to the <ship>.`
```

> 			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer the survivors to the <ship>.`

Reviewer 3: ```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms, and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer the survivors to the <ship>.`
```
The comma is there in the version of this line that appears if you have multiple crew, should probably be here too.

			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave their dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again. For the young, this gives them hope, but sometimes they are caught, and the Efreti are reminded why they live crushed together around these few stars."`
			`	Your communicator notifies you of a payment of <payment>.`
	npc
		system "Seketra"
		fleet "Small Kor Mereti"
		government "Kor Mereti"
		to spawn
			random < 30
		personality
			staying

mission "Efreti: Distress Call 3"
	name "Recover the fallen crew of the <EDC-ship>"
	description "Recover the bodies of the dead <EDC-ship> crew on <destination> and return them to <origin>."
	source "Laki Nemparu"
	destination "Seleptra Nak"

> 	cargo "Efreti coffins" 13

Author: Efreti is a proper noun, and should be capitalized wherever it appears. If other txt doesn't do that, it should be fixed, but not in this PR.

> 	cargo "Efreti coffins" 13

Author: It is lower-case now, because the game automatically capitalizes it when displaying it.

			choice
				`	"I will bring them home."`
					goto home
				`	"I will if they pay me."`
					goto pay
				`	"I won't risk my ship for the dead."`
			`	The leader of the group visibly sags as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
				decline

			label home
			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
				goto title

			label pay
			`	The group relaxes quickly as you see a ripple of relief and pride pass through them.`

			label title
			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "Efreti crews have families, as children learn by working with parents. For some families, three generations were lost. We will not try to salvage the ship, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
				accept
	npc

> 		system "Seketra"
> 		fleet "Small Kor Mereti"
> 		government "Kor Mereti"
> 		to spawn
> 			random < 30
> 		personality
> 			staying

Reviewer 4: ```suggestion
		government "Kor Mereti"
		personality staying
		system "Seketra"
		to spawn
			random < 30
		fleet "Small Kor Mereti"
```

mission "Efreti: Distress Call 4"
	landing
	name "Recover the fallen of <EDC-ship>"

> 	description "Return the bodies of the dead <EDC-ship> crew to <destination>."

Reviewer 5: ```suggestion
	description "Return the bodies of the dead <EDC-ship> crew to <planet>."
```

	source "Seleptra Nak"
	destination "Karek Fornati"

> 	cargo "Efreti deceased" 13

Author: Efreti is a proper noun and should be capitalized, regardless of errors elsewhere in Endless Sky txt.

> 	cargo "Efreti deceased" 13

Reviewer 6: While that is true, it is intentional we don't use capital words even if proper for data such as cargo lines. It's not a matter of the word being proper, it's a matter of how we format the data, in which this case it should be lower case.

> 	cargo "Efreti deceased" 13

Author: It isn't merely data if it is displayed to the user.

> 	cargo "Efreti deceased" 13

Reviewer 6: Regardless, if the convention is done the same way everywhere else, please do so in this PR. As you have said, if it should be fixed, please do so in another PR, and keep to the standard in this one, rather than make it confusing here.

> 	cargo "Efreti deceased" 13

Author: Oh! I see now. No matter how you capitalize it, the code automatically displays it capitalized in-game:

![Screenshot_2023-05-27_13-28-14](https://github.com/endless-sky/endless-sky/assets/116329264/f7dfd0e7-2415-427f-a2cc-e34d153de713)

	to offer
		has "Efreti: Distress Call 3: done"
	on offer
		conversation
			`Locating the <EDC-ship>, you and your crew enter it wearing environment suits. It's been several days since the crash, and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
				to display
					"flagship crew" > 1
			`Locating the <EDC-ship>, you enter it wearing an environment suit. It's been several days since the crash, and Korath ships run hot. This will be an unpleasant task in the suit and perhaps impossible without it, especially with no crew helping you.`
				to display
					"flagship crew" == 1

> 			`	While your engineer works on refreshing the atmosphere with outside air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`

Reviewer 2: Maybe they multitask.

> 			`	While your engineer works on refreshing the atmosphere with outside air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`

Author: In that paragraph, you have at least three crew members, including yourself. It is reasonable to expect that one of them has engineering experience.

For the paragraph that has only one crewmate, they're referred to as a "crewmate."

				to display
					"flagship crew" > 2
			`	While your crewmate works on refreshing the atmosphere with outside air, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
				to display
					"flagship crew" == 2
			`	One by one, you gather the bodies and place them in fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
				to display
					"flagship crew" < 2

> 			`	Returning to the Korath ship, you carefully gather all the personal effects you can find in the living quarters and box them for transport. You've no way of knowing if those. in what appear to be photos. are now safely recovering or lying in your hold.`

Reviewer: ```suggestion
			`	Returning to the Korath ship, you carefully gather all the personal effects you can find in the living quarters and box them for transport. You've no way of knowing if those Korath in the photographs you find are now safely recovering or lying in your hold.`
```

			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch, and you count the coffins together with them. Afterwards, you show them the boxes of possessions, which they handle with care. An officer thanks you and hands you a data chip that vaguely reminds you of a license.`
			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, presumably their family.`
			`	Rather than intrude on their grief, you return to your ship, slotting the data chip you received into your systems. Unfortunately, the information is in an alien language, so you set it aside and work on a plan to salvage the <EDC-ship>. Gathering some notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage it. The closest system that comes to mind with many human speakers is on Greenwater, in Hai space.`

> 	npc
> 		system "Seketra"
> 		fleet "Small Kor Mereti"
> 		government "Kor Mereti"
> 		to spawn
> 			random < 30
> 		personality
> 			staying

Reviewer 4: ```suggestion
	npc
		government "Kor Mereti"
		personality staying
		system "Seketra"
		to spawn
			random < 30
		fleet "Small Kor Mereti"
```
Again, above the `on complete`.

mission "Efreti: Distress Call 5"
	name "<EDC-ship> salvage team"
	description "Hire a <bunks> member salvage team on <destination> to salvage the Efreti ship <EDC-ship>."
	source "Karek Fornati"
	destination "Greenwater"
	landing
	passengers 66
	
	to offer
		has "Efreti: Distress Call 4: done"
	
	on complete
		conversation
			`Many stories tell of Hai who like change and excitement, but their enthusiasm seems to have died. Thoughts of the species that invaded them still inspire fear. No Hai have the courage to salvage a ship that has been caught in the crossfire of the Mereti and Sestor war.`

> 			`	Fortunately, there are plenty of Humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`

Reviewer 4: This mission requires passenger space, but has no blocked message, and no `on offer` conversation.

I suggest moving the `on complete` conversation from the previous mission into the `on offer` of this one, removing the passengers from this, and move the `on complete` conversation from this mission the `on offer` of the next one.

Reviewer 2: ```suggestion
mission "Efreti: Distress Call 5"
	name "<EDC-ship> salvage team"
	description "Hire a <bunks> member salvage team on <destination> to salvage the Efreti ship <EDC-ship>."
	source "Karek Fornati"
	destination "Greenwater"
	landing
	passengers 66
	blocked `You need <bunks> available to carry the salvage team.`
	
	to offer
		has "Efreti: Distress Call 4: done"
	
	on complete
		conversation
			`Many stories tell of Hai who like change and excitement, but their enthusiasm seems to have died. Thoughts of the species that invaded them still inspire fear. No Hai have the courage to salvage a ship that has been caught in the crossfire of the Mereti and Sestor war.`
			`	Fortunately, there are plenty of humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`
```

Is this `blocked` message good? (Also, I think human should be lowercase)

ship "Charm-Shallop" "Charm-Shallop (Distressed)"
	outfits
		"Digger Mining Beam" 2
		"Fuel Processor"
		"Generator (Furnace Class)"
		"Large Heat Shunt"
		"Plasma Core"
		"Scram Drive"
		"Small Heat Shunt"
		"Steering (Planetary Class)"
		"Thermal Repeater Rifle" 19
		"Thruster (Lunar Class)"
		"Warder Anti-Missile"

> 	turret 0 -35 "Warder Anti-Missile"
> 	turret 0 97
> 	gun 0 -118 "Digger Mining Beam"
> 	gun -16.5 -114.5 "Digger Mining Beam"
> 	gun 16.5 -114.5

Reviewer 4: ```suggestion
	gun 0 -118 "Digger Mining Beam"
	gun -16.5 -114.5 "Digger Mining Beam"
	gun 16.5 -114.5
	turret 0 -35 "Warder Anti-Missile"
	turret 0 97
```
Guns before turrets.

mission "Efreti: Distress Call 6"

> 	name "Salvage the <EDC-ship>"
> 	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
> 	source "Greenwater"
> 	destination "Seleptra Nak"
> 	landing

Reviewer 4: ```suggestion
	landing
	name "Salvage the <EDC-ship>"
	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
	source "Greenwater"
	destination "Seleptra Nak"
```

	passengers 66
	blocked "You need to be able to carry <bunks> additional crew to salvage the ship. Return here when you have sufficient crew capacity."

	to offer
		has "Efreti: Distress Call 5: done"

> mission "Efreti: Distress Call 6 Ships"
> 	invisible
> 	landing
> 	source "Greenwater"
> 	to offer
> 		has "Efreti: Distress Call 5: done"
> 	to fail
> 		or
> 			has "Efreti: Distress Call 7: failed"
> 			has "Efreti: Distress Call 7: aborted"
> 			has "Efreti: Distress Call 7: done"
> 	npc kill
> 		system "Seketra"
> 		fleet "Small Kor Mereti" 2
> 		government "Kor Mereti"
> 		personality
> 			staying

Reviewer 4: ```suggestion
mission "Efreti: Distress Call 6 Ships"
	landing
	invisible
	source "Greenwater"
	to offer
		has "Efreti: Distress Call 5: done"
	to accept
		has "Efreti: Distress Call 6: active"
	to fail
		or
			has "Efreti: Distress Call 7: failed"
			has "Efreti: Distress Call 7: aborted"
			has "Efreti: Distress Call 7: done"
	npc kill
		government "Kor Mereti"
		personality staying
		system "Seketra"
		fleet "Small Kor Mereti" 2
```

mission "Efreti: Distress Call 7"
	landing
	name "Salvage the <EDC-ship>"
	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
	source "Seleptra Nak"
	destination "Karek Fornati"

> 	landing

Reviewer 4: ```suggestion
```
There's already a `landing` at the top of this mission.

	to offer
		has "Efreti: Distress Call 6: done"
	on offer
		conversation
			`Thanks to the work of your crew, the atmosphere on the ship is brought to human standards, if a bit tropical. As the salvage continues, you and your crew start to become familiar with the ship. Its strong hull appears spaceworthy, but the ship has barely any weapons, the battery systems are beyond repair, and shield regeneration is impossible. If the Mereti attack, the <EDC-ship> won't be able to survive long.`
			choice
				`	(Fly the <EDC-ship> home.)`
					goto runforit
				`	(Abandon the ship.)`
					goto abandon
			
			label abandon
			action
				fail

> 			`	Your crew abandons the Korath ship to the elements. It's just one more tragic story amid the ruins of their civilization.`

Reviewer: ```suggestion
			`	Your crew abandons the Korath ship to the elements, just one more tragic story amid the ruins of their civilization.`
```

				flee
			
			label runforit
			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
				launch

	on complete

> 		fail "Efreti: Distress Call 6 Ships"

Reviewer 4: If you're going to fail it here, don't put the auto condition set by this action (`[...]: done`) in the `to fail` of that mission. It's just confusing for anyone reading it.

Author: I will try removing this one and see if it works.

Author: It works.

			choice
				`	(Drink it.)`
					goto drink
				`	(Make a motion as if drinking.)`
			`	Either the Efret don't notice your ruse, or they are too polite to say. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
				goto end
			label drink
			`	The drink is fermented, and has a sweet and sour taste that leaves your lips and mouth tingling, eventually numbing them. The Korath then presses the sweets on you, and you humbly accept them. A few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
			label end
	on visit
		dialog "You arrive, but you notice the <EDC-ship> is absent. You'll have to bring it here to finish the salvage mission."

> 	npc save accompany

Reviewer 4: ```suggestion
	npc accompany save
```

		# Ship name should match the <EDC-ship> substitution above.
		government "Kor Efret"
		personality timid escort

> 		ship "Charm-Shallop (Distressed)" "Esketari"

Reviewer 4: Move this block above the `on complete`.

mission "Efreti: Distress Call 8"

> 	invisible
> 	landing

Reviewer 4: ```suggestion
	landing
	invisible
```

	source
		attributes quarg
	to offer
		has "Efreti: Distress Call 7: done"
	on offer
		conversation
			`The Quarg have translated the Korath language for you in the past. You have the data chip from when you helped the Efret salvage the <EDC-ship>. Would you like to see if the Quarg can tell you what the data chip says?`
			choice
				`	(Ask the Quarg to translate the data chip.)`
				`	(Maybe another time.)`
					defer
				`	(I don't care what it says.)`
					decline
			
			`	You approach one of the Quarg in the spaceport and request assistance with the data chip. It stops and acknowledges you, but doesn't speak. It pauses for a few moments after you make your request, but then holds out its hand. Taking the data chip, it slots it into a communicator of some sort, slowly nods to itself, and hands it back to you.`

> 			# The message is translated from Efreti. Korath languages have no tenses due to their biology. It was translated by Quarg, who use archaic writing and wording. Hence the odd writing style.

Reviewer 4: We don't have (or need) comments on every line of alien speech that doesn't conform to the way humans speak.
```suggestion
```

Author: I add these for the sake of reviewers. They should all be removed before merge.

Author: The four comments that were there for the sake of reviewers are gone. I hope I won't have to put them back in.

		# Ship name should match the <EDC-ship> substitution above.
		government "Kor Efret"
		personality timid escort
		ship "Charm-Shallop (Distressed)" "Esketari"

mission "Efreti: Distress Call 8"
	invisible
	landing
	source
		attributes quarg
	to offer
		has "Efreti: Distress Call 7: done"
	on offer
		conversation
			`The Quarg have translated the Korath language for you in the past. You have a data chip from when you helped the Efret salvage the <EDC-ship>. Would you like to see if the Quarg will tell you what the data chip says?`
			choice
				`	(Ask the Quarg to translate the data chip.)`
				`	(Maybe another time.)`
					defer
				`	(I don't care what it says.)`
					decline
			
			`	You approach one of the Quarg in the spaceport. It stops and acknowledges you but doesn't speak. It pauses for a few moments after you make your request but then holds out its hand. It takes the data chip and slots it into a communicator of some sort. It then nods to itself and hands you back the chip.`
			# The message is translated from Efreti. Korath languages have no tenses due to their biology. It was translated by Quarg, who use archaic writing and wording. Hence the odd writing style.

> 			`	You plug the chip into your own device and see two files: one in the Korath language, one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`

Reviewer 6: ```suggestion
			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
```
The previous time you attempted to read the chip, you slotted it into your ship. Adjusted wording for that, instead of a magical device you happened to have had on you.

> 			`	You store the chip carefully and return to your ship.`

Reviewer 6: ```suggestion
			`	You carefully store the chip in a safe place.`
```

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

substitutions
	"<EDC-ship>" "Esketari"

mission "Efreti: Distress Call 1"
	minor
	landing
	name "Distress call from the <EDC-ship>"
	description "The Quarg have asked you investigate a distress call from the Efreti ship <EDC-ship> which has crashed on <destination>."
	source "Karek Fornati"
	destination "Seleptra Nak"

> 	passengers 40

Author: I have updated the mission description to imply that.

			`	He leads you into the ship, shouting into a communication device that is then picked up on speakers throughout the ship. You can only glance around as you move through the engineering section but it's clear they are running on emergency power. You notice that their Systems Core was destroyed, leaving them stranded.`
			`	The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree with more than half incapacitated or unconscious. Even if the ship is spaceworthy, there aren't enough crew to fly it.`
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`

> 				to display

Author: The only text in that mission that speaks of your crew number is that one. The `accept` prevents the dialog from continuing to the next line.

> 				to display

Author: There is a comment explaining this now.

			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave their dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
			`	Your communicator notifies you of a payment of <payment>.`
	npc
		system "Seketra"
		fleet "Small Kor Mereti"
		government "Kor Mereti"
		to spawn
			random < 30
		personality
			staying

mission "Efreti: Distress Call 3"
	name "Recover the fallen crew of the <EDC-ship>"
	description "Recover the bodies of the dead <EDC-ship> crew on <destination> and return them to <origin>."
	source "Laki Nemparu"
	destination "Seleptra Nak"
	landing

	to offer
		has "Efreti: Distress Call 2: done"

> 

Author: Actually, it should have cargo. Efreti are not weightless incorporeal beings.

				accept
	npc
		system "Seketra"
		fleet "Small Kor Mereti"
		government "Kor Mereti"
		to spawn
			random < 30
		personality
			staying

mission "Efreti: Distress Call 4"
	landing
	name "Recover the fallen of <EDC-ship>"
	description "Return the bodies of the dead <EDC-ship> crew to <destination>."
	source "Seleptra Nak"
	destination "Karek Fornati"
	to offer
		has "Efreti: Distress Call 3: done"
	on offer
		conversation
			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
				to display
					"flagship crew" > 1
			`You enter the <EDC-ship> wearing an environment suit. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suit and perhaps impossible without it, especially with no crew helping you.`
				to display

> 					"flagship crew" <= 1

Author: I prefer accounting for possible bugs, such as `flagship crew` malfunctioning.

	on complete
		payment 1000
		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`

substitutions
	"<EDC-ship>" "Esketari"

mission "Efreti: Distress Call 1"
	minor
	landing
	name "Distress call from the <EDC-ship>"
	description "The Quarg have asked you investigate a distress call from the Efreti ship <EDC-ship> which has crashed on <destination> with <bunks> survivors."
	source "Karek Fornati"
	destination "Seleptra Nak"
	passengers 40
	blocked "A Quarg ship broadcasts that they need a ship with at least <bunks> bunks for an emergency evacuation."

	to offer
		not "event: wanderers: kor mereti friendly"
		has "First Contact: Karek Fornati: offered"
		random < 30
		"combat rating" > 250
		
	on offer
		conversation
			`The blue flash of Quarg ships is nothing new as you travel Efreti space. Usually that is all you see of the enigmatic aliens. Today, however, you receive an urgent hail as you land.`

> 			`	"Salutations Captain of the <ship>. Your most diligent efforts are required lest many families mourn the dead. An Efreti ship, the <EDC-ship>, was surveying <planet> when surprised by Mereti they were. Their ship crashed on the planet with many dead and they have asked for help to evacuate."`

Author: I merged this change, but I left the conversation unresolved since you're asking for further discussion.

> 			`	"Salutations Captain of the <ship>. Your most diligent efforts are required lest many families mourn the dead. An Efreti ship, the <EDC-ship>, was surveying <planet> when surprised by Mereti they were. Their ship crashed on the planet with many dead and they have asked for help to evacuate."`

Author: I rewrote the second paragraph so it doesn't sound like Yoda anymore.

			choice
				`	"I'll go there immediately."`
					accept
				`	"Can't the Quarg respond?"`
					goto quarg
				`	"I don't care about the Korath."`
					goto careless
			
			label quarg

> 			`	"We vanquished the Mereti, but the <EDC-ship> fell beyond our reach and crashed. Strong armor we need to survive the pressure of a heavy world to help the Korath, while a human would have few problems. You can reach them days before we could return from Kuwaru Efreti."`

Author: I have un-Yodad the speech and made it sound archaic.

	npc
		system "Seketra"
		fleet "Small Kor Mereti"
		government "Kor Mereti"
		to spawn
			random < 30
		personality
			staying

mission "Efreti: Distress Call 3"
	name "Recover the fallen crew of the <EDC-ship>"
	description "Recover the bodies of the dead <EDC-ship> crew on <destination> and return them to <origin>."
	source "Laki Nemparu"
	destination "Seleptra Nak"
	cargo "Efreti coffins" 13
	landing

	to offer
		has "Efreti: Distress Call 2: done"

	on offer
		conversation
			`You are relaxing at a spaceport restaurant now that the evacuation is complete when a delegation of Korath approach you somewhat hesitantly. They are dressed in what appears to be much finer clothing than most of the Korath you have met, certainly nothing you would wear to do work in a spaceport. As they make the traditional gesture of greeting, which you return, you are curious both as to what they want and how they will ask.`
			`	One of them takes a small case out of a pocket and removes a black device that looks like three triangles hooked together. He hisses at it for a moment and then it rises out of his hand. The air around it begins to shimmer and soon a life size hologram of a Quarg appears.`

> 			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan-company that you have assisted. They thank you for your aid in saving their children and grandchildren. While most grateful they are for the living, they cannot fully mourn the dead who lie on <planet>. They ask me to ask that you travel there and bring their lost ones home to <planet>."`

Reviewer 5: ```suggestion
			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan-company that you have assisted. They thank you for your aid in saving their children and grandchildren. While most grateful they are for the living, they cannot fully mourn the dead who lie on <planet>. They ask me to ask that you travel there and bring their lost ones home to <origin>."`
```

> 			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan-company that you have assisted. They thank you for your aid in saving their children and grandchildren. While most grateful they are for the living, they cannot fully mourn the dead who lie on <planet>. They ask me to ask that you travel there and bring their lost ones home to <origin>."`

Author: I committed this suggestion, but you asked for further comment, so I left this unresolved.

			choice
				`	"I will bring them home."`
					goto home
				`	"I will if they pay me."`
					goto pay
				`	"I won't risk my ship for the dead."`
			`	You see the leader of the group visibly sag as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
				decline

			label home
			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
				goto title

			label pay

> 			`	A ripple of what appears to be dissatisfaction passes through the group, but their leader silences them with a gesture.`

Reviewer 5: On the previous PR, I discussed that the Ko Efreti have previously been characterized as a proud species insistent on paying. `dissatisfaction` upon the player mentioning payment seems to go against that characterization. Could we come up with some alternatives for this line?

Author: I have rewritten it accordingly. How does it look now?

			label title

> 			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "For many ship skills, parent and child work to learn. For some families, three generations are lost. We will not try to salvage the body box of our kin, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`

Reviewer 5: From the previous PR as well - this might be a little _too_ rough on Quarg translation. I think the player should still be able to understand what the Korath are offering (permission to take the ship). This line may also need to change pending the conversation around whether the ship will be rewarded to the player.

Author: Actually, I did improve it a bit. However, I just improved it some more. Do you like it?

Author: I rewrote that paragraph again.

There are internal Korath developer discussions going on about how to handle tenses when Korath are giving instructions. Korath languages don't have past or future tenses like we do, mainly for biological reasons. (This would apply to all Korath, including the Efreti.) When they give instructions, they need to express a progression of events and causality. We don't have a final answer yet as to how that should be presented. This is waiting on ponderings from both Ravenshining and I, and future discussions on Swizzle6.

This is a Quarg translation of a Efreti, which is a bit of an odd situation. It may be acceptable for the Quarg to add in tenses, by providing its interpretation of the information. However, because it is a Quarg, I modified the text to sound archaic.

In other words, you're getting an archaic Quarg translation of an Efreti's tense-less instructions.

				accept
	npc
		system "Seketra"
		fleet "Small Kor Mereti"
		government "Kor Mereti"
		to spawn
			random < 30
		personality
			staying

mission "Efreti: Distress Call 4"
	landing
	name "Recover the fallen of <EDC-ship>"
	description "Return the bodies of the dead <EDC-ship> crew to <destination>."
	source "Seleptra Nak"
	destination "Karek Fornati"
	cargo "Efreti deceased" 13
	to offer
		has "Efreti: Distress Call 3: done"
	on offer
		conversation

> 			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`

Reviewer 6: ```suggestion
			`Locating the <EDC-ship>, you and your crew enter it wearing environment suits. It's been several days since the crash, and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
```

Changing excessive use of "You...", and this helps lead into the situation rather than you landing on the planet and are suddenly entering the ship, especially when first locating it was difficult. Adding the crew part is important when referring to "suits".

				to display
					"flagship crew" > 1

> 			`You enter the <EDC-ship> wearing an environment suit. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suit and perhaps impossible without it, especially with no crew helping you.`

Reviewer 6: ```suggestion
			`Locating the <EDC-ship>, you enter it wearing an environment suit. It's been several days since the crash, and Korath ships run hot. This will be an unpleasant task in the suit and perhaps impossible without it, especially with no crew helping you.`
```
Ditto.

				to display

> 					"flagship crew" = 1

Reviewer 6: ```suggestion
					"flagship crew" == 1
```

			`	While your engineer works on refreshing the atmosphere with outside air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
				to display
					"flagship crew" > 2
			`	While your crewmate works on refreshing the atmosphere with outside air, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
				to display
					"flagship crew" == 2

> 			`	One by one, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`

Reviewer 6: ```suggestion
			`	One by one, you gather the bodies and place them in fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
```
Another case of using "the" for something not explained previously.

				to display
					"flagship crew" < 2

> 			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`

Reviewer 6: ```suggestion
			`	Returning to the Korath ship, you carefully gather all the personal effects you can find in the living quarters and box them for transport. You've no way of knowing if those. in what appear to be photos. are now safely recovering or lying in your hold.`
```
Adding this makes it more interesting and also helps alleviate the sudden "there are photos" without explaining that. These are aliens, let it be a bit less friendly to direct comparisons.

			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
				to display
					"flagship crew" > 1
			`	As a precaution, you make a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
				to display
					"flagship crew" <= 1
			branch accept
	on complete
		conversation

> 			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch. You count the coffins together and show them the boxes of possessions.`
> 			`	The officer thanks you and hands you a data chip that vaguely reminds you of a license.`
> 			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, you presume their family.`
> 			`	You go back inside your ship rather than intrude on their grief. You slot the data chip into your systems but the information is in an alien language. You set it aside and work on a salvage plan.`
> 			`	As you review your notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage the <EDC-ship>. The closest system where you can find that many human speakers is on Greenwater in Hai space.`

Reviewer 6: ```suggestion
			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch, and you count the coffins together with then, then show them the boxes of possessions. An officer thanks you and hands you a data chip that vaguely reminds you of a license.`
			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, presumably their family.`
			`	Rather than intrude on their grief, you return to your ship, slotting the data chip you received into your systems. Unfortunately, the information is in an alien language, so you set it aside and work on a plan to salvage the <EDC-ship>. Gathering some notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage it. The closest system that comes to mind with many human speakers is on Greenwater, in Hai space.`
```
As with earlier, too much mechanical split-up. Editing the last line to be more impactful on coming up with the idea, rather than just trailing onto it casually.

I'm stopping my first review here - this should be barred if you haven't made contact with the Hai yet. Either the *whole* mission series should be barred based on that, or there should be an alternative to going to the nearest human planet in human space if you haven't met the Hai.

Author: It is not possible to check if Greenwater has been *mapped*. You can check if the system was entered, but that's slightly unfair since the player has no way to know if they've ever entered the system.

I could do a PR to add the capability of checking if a system has been mapped, but then we'd be waiting on another PR. I think not having mapped Greenwater is a small enough niche case that it can be ignored in this PR for now.

Reviewer 6: Not only can you check if a system was entered, but you can check if a player has visited a planet.

Author: That isn't fair either. You just need to know the planet exists and is Hai. For that, the minimum is that it must be mapped. If you found it by purchasing a map, then you won't have ever entered it.

Reviewer 6: Oh, we do have it.

`"visited planet: <planet>"`

Reviewer 6: I'm not sure what you're suggesting by "mapping" then.

Author: Actually, it turns out there are no outfitters that can map Greenwater, so this niche case possible anyway. It is sufficient to have `visited system: Fah Soom`. I have added that requirement.

Also, I was wrong about what HasSeen does. It just checks if you've seen the system, regardless of whether you know the name or contents.

Author: > I'm not sure what you're suggesting by "mapping" then.

You can put a system on your map two different ways:

1. Flying into the system.
2. Buying a map from an outfitter.

It turns out there are no outfitters that will sell you a map to Fah Soom.

			choice
				`	(Fly the <EDC-ship> home)`
					goto runforit
				`	(Abandon the ship)`
					goto abandon
			
			label abandon
			action
				fail
			`	Your crew abandons the Korath ship to the elements. It's just one more tragic story amid the ruins of their civilization.`
				flee
			
			label runforit
			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
				launch

	on complete
		fail "Efreti: Distress Call 6 Ships"
		conversation
			action
				# Ship name should match the <EDC-ship> substitution above.
				give ship "Charm-Shallop (Distressed)" "Esketari"
			`There are no crowds waiting for you when you land; rather, the ground crews seem astonished to see the ship. It takes you some time to arrange for repairs. By the time you're finished, you see a pavilion has been set up near the <EDC-ship> and small groups of Korath periodically stop by.`
			choice
				`	(Check it out.)`
				`	(I've got places to be.)`
					decline
				

> 			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`

Reviewer 5: In the previous PR, Will made the point that `robe` is usually referred to in the plural (`robes`). to me, it still feels weird given the singular nature of the characterization right before the word (`in some kind of`). I have two solutions; up to you which to pick.

```suggestion
			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in robes stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`
```
OR
```suggestion
			`	The tent is larger than you expected. At the back there is a group of Korath arranged like a choir, but standing oddly still. A Korath in some kind of robe stands next to a table piled with sweets and cups of some kind of thick drink. As you approach, you realize that the "choir" is made of holograms. You do a quick mental count and guess that these are the Korath that died on the ship; this is probably some type of memorial. The Korath in robes greets you and presses a cup into your hand.`
```
